### PR TITLE
Added Trillian Beta for Mac

### PR DESCRIPTION
**Pull request by N4M3Z**
Added Trillian Beta for Mac

### CODE REVIEW

File: Casks/trillian-beta.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'trillian-beta' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/trillian-beta.rb]: It's missing `sha256`, which is required. The cask should be:

```ruby
cask :v1 => 'trillian-beta' do
  version :latest
  sha256 :no_check

  url 'https://www.trillian.im/get/mac/beta/'
  name 'Trillian'
  homepage 'https://www.trillian.im'
  license :commercial

  app 'Trillian.app'
end
```

**Comment by vitorgalvao** [comment]
It's missing `sha256`, which is required. The cask should be:

```ruby
cask :v1 => 'trillian-beta' do
  version :latest
  sha256 :no_check

  url 'https://www.trillian.im/get/mac/beta/'
  name 'Trillian'
  homepage 'https://www.trillian.im'
  license :commercial

  app 'Trillian.app'
end
```

**Comment by N4M3Z** [comment]
Sorry, done, just trying to help

**Comment by vitorgalvao** [comment]
I know, and the contribution is very welcome. That wasn’t meant to be confrontational, if that is what it seemed; quite the contrary, simply explanatory.

**Pull request merged**